src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -59,6 +59,14 @@
 }
 
 
+def _validate_stage_path(path: str) -> str:
+    if not path.strip("\"'").startswith("@"):
+        raise ValueError(
```

Comment:
do we also need to take care of snow urls `snow://`?

Reply:
I'm unfamiliar with those. Is there good documentation somewhere?